spec/requests/api/v1/driver_commission_histories_spec.rb

```diff
@@ -10,15 +10,15 @@
   end
 
   it 'responds with all driver commission history json' do
-    get '/api/v1/driver_commissions_history'
+    response = doorkeeper_token.get('/api/v1/driver_commissions_history')
 
     expect(response.body).to eq(all.to_json)
   end
 
   it 'responds with driver commission history by paid date json' do
-    # rubocop:disable
-    get '/api/v1/driver_commissions_history_by_paid_date.json?paid_date=2012-01-01'
-    # rubocop:enable
+    # rubocop:disable Metrics/LineLength,
+    response = doorkeeper_token.get('/api/v1/driver_commissions_history_by_paid_date.json?paid_date=2012-01-01')
```

Comment:
Unfortunately the request itself is too big for the 83 char limit set in .rubocop_todo.yml with the indentation needed. :(